### ISSUE TITLE
Create deprecation plan for the "Gutenberg" name

### ISSUE THREAD

**Issue by mor10** [WordPress/gutenberg#4681]
## Issue Overview

tl;dr: "Gutenberg" is a fine name for this project, but unless we stop using it now, it'll become a point of great confusion once the feature is merged with core.

As blocks and block editing move closer to maturity and core merge, a plan is needed for how to get the community to think of these as core features, not add-on features included theough a plugin named for an old German goldsmith. This is a matter of some urgency as the names "Gutenberg" and "Gutenblocks" is being used for and applied to articles, tutorials, documentation, and solutions including plugins and themes that will live on long after the core merge.

Once blocks are a core feature, the name "Gutenberg" will be a source of confusion in the community, further complicated by the many other "Gutenberg" projects available online.

I propose a deprecation plan be put in place immediately to start offloading the "Gutenberg" name in favor of something more descriptive that will make sense once the feature is in core. "Block editor" or "WordPress blocks" or something similar. An outreach campaign is also needed to get influencers in the community to use the new name over "Gutenberg" in their articles, tutorials, presentations, and products. A stop-gap measure could be something like encouraging the use of "WordPress blocks, formerly Gutenberg" or an equivalent.

Clear language matters. This is the time to take steps to ensure language around block editing in WordPress is clear. No offense to Johannes.

**Comment by jdevalk** [comment]
I say we call it Coster. After [Laurens Janszoon Coster](https://en.m.wikipedia.org/wiki/Laurens_Janszoon_Coster).

**Comment by lushkant** [comment]
Agreed Totally!!
Saw earlier today many people having conversations in a Linkedin group "what Gutenberg is and what does it really stand for".
What sort of name it's if it doesn't make sense to the end users. I mean if it's only a codename for the ones who're developing it then they're better calling it with id's.

 ( Just a close watcher, not a big deal )..

**Comment by bobbingwide** [comment]
The plan needs to consider the names for each of the different editors that a user may be able to use, wherever they happen to be in the WordPress site. 

1. Gutenberg Editor - Visual Editor 
2. Gutenberg Editor - Code Editor
3. Classic Editor - Visual tab - Rich editing
4. Classic Editor - Text tab - 
5. Widgets - Text
6. Widgets - Custom HTML 
7. Theme & plugin editor
8. Customizer
9. Other

The convention should take into account the user's WordPress experience.

Should this plan also take into account refactoring of the code? function names, comments, CSS styling, links etc?

**Comment by schlessera** [comment]
I think the most obvious approach would be to let people know that "The visual editor has been revamped!" and that "You can now use draggable blocks of content to shape your page!".

From the user's point view, there's no reason for it to have a proper name - the editor has changed, that's all. That it is, technologically-speaking, a completely different approach and goes far beyond the actual editor is something the developers should be aware of, but other than that, it is completely inconsequential for an end user trying to write content.

**Comment by mor10** [comment]
@schlessera Exactly: The "Gutenberg" name is meaningless to the people who will be using blocks to build content. If it's not deprecated now before launch, it will live on in the ecosystem as a confusing term without any real context. 

Think about it this way: When blocks are in core and people go to WordPress TV to learn about them, they won't search for "Gutenberg" even though that's where all the information currently resides. Same with almost all tutorials etc. Plugins also ship "Gutenberg ready" content and "Gutenblocks" and use these terms to market themselves. Again, this will not work once blocks are in core because the end-users will not search for the term "Gutenberg" since it has no meaning to them.

**Comment by carlhancock** [comment]
@mor10 Is absolutely correct.  Gutenberg is fine for a project codename while in development... but it needs to cease to exist when rolled into core.  It won't be Gutenberg anymore.  It will simply be the WordPress editor.

This has implications beyond what it's called when the code is rolled into core... it means any and all documentation for it needs to not call it Gutenberg.  It means any development resources that teach people how to develop for it needs to not call it Gutenberg.  It also means training resources like @zgordon's Gutenberg Development Course would need to be updated and rebranded.

When Gutenberg is rolled into core it will cease to be Gutenberg and will simply be WordPress.  I agree that continuing to refer to it as Gutenberg after that occurs will simply cause user confusion.

**Comment by zgordon** [comment]
Great discussion! Within my course I try to just reefer to things by actual names like block or editor so it will be more future compatible. 

My plan is definitely to adapt the name of the course at some point, possibly when everything rolls into Core.

That said, at a practical level, since Gutenberg has been the name used for some time now in articles and talks I imagine there will be some overlap between when things roll into Core and when people stop saying things like 'Gutenberg Ready' 

That said I'm totally behind this transition and thanks Morten for bringing it up so we can all do this together at the same time and in the same way.

**Comment by greatislander** [comment]
I think the existence of the (entirely unrelated) Project Gutenberg has the potential to confuse some users. Thanks for raising this issue, @mor10.

**Comment by duanestorey** [comment]
@greatislander I have a post about Gutenberg (WordPress) on my site, and half the traffic that hits it is searching for Project Gutenberg (not WordPress), so there is already some confusion.

**Comment by mor10** [comment]
@greatislander not to mention several other web related projects with the same name including https://github.com/matejlatin/Gutenberg, https://www.npmjs.com/package/gutenberg, and an entire academic field around the person Johannes Gutenberg and the printing press.

@duanestorey This is precisely why a plan needs to be put in place now and enacted immediately. Insiders are using the term to get people to read/watch/download their stuff. This is fracturing the available information and will cause confusion down the road. This is one of those cases where being aggressively proactive, even if it's frustrating at first, is essential.

**Comment by voldemortensen** [comment]
Counterpoint: If memory servers me well, @m mentioned a few times that Gutenberg is a separate project that will be bundled into core, but also exist separately so it can be integrated into other projects. I agree that in core and in WordPress related documentation referring to it as Gutenberg will cause confusion for users, but once this is "rolled into core", this repository will continue to be maintained separately (unless I am mistaken). Dropping the Gutenberg name entirely could lead to lack of adoption, contributors, etc.

**Comment by mor10** [comment]
@voldemortensen Counterpoint point: I don't think that's a counterpoint to my point. If Gutenberg is going to live as a project separate for WordPress (which it currently is not), it should be branched off and divorced from WordPress functionality and renamed because for the rest of the world, the "Gutenberg" name is strongly associated with the Gutenberg Project (books, not WordPress). In any case, the status of Gutenberg as a stand-alone project does not change the proposal here: In the WordPress context, the terminology needs to change now to avoid confusion and fracturing of information in a couple of months.

**Comment by voldemortensen** [comment]
@mor10 If it's not going to live separately, then I think we're on the same page. The proposal, to me, didn't read as specifically for the WordPress context (I could have missed something). In any case, you've clarified that it is for the WordPress context, so my point is moot.

**Comment by ntwb** [comment]
_MP6 is rarely heard of anymore #justsayin_

**Comment by mor10** [comment]
@ntwb MP6 was hardly ever talked about as "MP6" except for a few mentions by Matt. The "Gutenberg" name has entrenched itself as in-speak in the community.

**Comment by JJJ** [comment]
Keeping it named Gutenberg is probably what’s going to happen; that’s my prediction.

* It pays homage to a real life impactful person, which is hard to take away
* Phrases like `old_editor` and `new_editor` are obviously short sighted (tinyMCE has teeny, but can you _really_ tell me what it does without looking?)
* We’ve named themes after people
* We name releases after people, and it brings education and awareness
* The name becomes more than it was thanks to semantic change; a year from now only a few people will still choose to be bothered by it

### My personal opinions?

I think having action/filter/class/function names with `gutenberg` or `gb_` in them is about as good an idea as Apple having `ns_` in random stuff.

* It’s annoying and inconsistent
* It seems amateurish, like we should know it’s not ideal by now
* It will invoke good or bad memories years from now that have little to do with the thing
* It’s forever, whatever we pick, and keeping it named Gutenberg isn’t the best idea, but it’s not the worst

I’d rather see the overall name issue worked-around just using `block` for everything, especially since it’s unlikely that WordPress will ever include any type of “block” functionality beyond marking things as spam.

The flexibility of WordPress is due largely in part to its ambiguity. Being too targeted limits flexibility. It’s hard to see a Menu for anything than a Menu, but we spent 7 years making do with hierarchical Pages, many still do, and that’s still how it works under the hood.

TL;DR - I prefer we remove references in code, but not in spirit.

**Comment by ClintonGallagher** [comment]
Either way people will continue to refer to the Gutenberg Project as Gutenberg.
So take your pick...

_My Mama said "Stupid is what Stupid does."_
-- Forrest Gump

_My momma always said, "Life was like a box of chocolates. You never know what you're gonna get."_
-- Forrest Gump

**Comment by jdevalk** [comment]
All kidding above aside; I agree with Morten and vehemently disagree with JJJ. The issue is, we've never before taken charge of the narrative. We've never thought enough about these things. We should. They reflect on how mature as a project we are.

There are lots of things we should have done in our marketing and branding that we haven't. 2018 is going to be the year we change that. Because we *can* choose to; so we choose to.

Personally I'd vote for calling it "the editor", and when we have to compare it to the classic editor, referring to it as "the modern editor". 

Blocks are blocks or WordPress blocks. Seems simple enough.

**Comment by duanestorey** [comment]
@jdevalk I agree, but I'd be inclined to always call Gutenberg 'the editor' going forward, and reference the old version as the 'legacy editor'.  My thoughts on this are if you are having a conversation that involves post 5.0 and pre 5.0 users (i.e. us old folks), suddenly switching between talking about the 'editor' and the 'modern editor' may be confusing to the post 5.0 users, whereas I think legacy has the intended meaning (or at least, something in the past) even for people who may not even know what it is. With that in mind I'd propose the opposite, than the Gutenberg editor, once integrated into core, is 'the editor', and the old version is the 'legacy editor' or something to that affect.

**Comment by jdevalk** [comment]
Well modern is literally opposed to classic editor, which the plugin for the "old" editor is called. Of course thar can be changed, but THAT has been communicated already... And legacy is slightly harder for non-native speakers.

At the same time I think it's more important that we choose *something* and stick with it.

**Comment by JJJ** [comment]
To your own point, what’s the next editor called 12 years from now? The more-modern editor? The even-newer editor?

It’s just the-editor, since it’s the default interface. To your point about being mature, calling it something binary and closed-ended seems terribly short-sighted.

To be clear, there are at least 2 different concerns: code, marketing.

* The code shouldn’t have Gutenberg references IMO, but it’s not the end of the world if it does. I’d like to avoid it, but it’s inevitably human to leave it as is, or risk abandoning the support of all of the early adopters.
* The marketing is already done to our inner circle of people as a code name; code names pretty much live on forever in our hearts, and nobody else’s, and that’s OK.

Happy to be disagreed with. Let’s get this solved sooner than later so we can put our efforts towards bigger problems.

**Comment by mikeschinkel** [comment]
Ah man, there goes the marketing for a line of plugins I had planned to brand under the name  _”Guten-Free!”_ :-D

**Comment by ahmadawais** [comment]
🙌 

After reading the story of both sides, I have to say that every single one of you has made valid points here.
But here's what I think!

🤔 I think then name `Gutenberg` should not be depreciated! It should stick! The name Gutenberg means something by now, calling it simple `the editor` or `the new editor` or `the blocks editor` or `the modern editor` (modern for how long? next ten years? and then...) or simple `blocks` will not be enough.

🔰 Calling it `Gutenberg` means something and it should. It's different, a bold new WordPress improvement. If you think that deprecating that name will bring less confusion to the table then that's where I disagree with you. Calling the new themes and plugins `Gutenberg` compatible means something!

And I have many practical examples here. 💯

#### 📦 Example #1 [create-guten-block](https://github.com/ahmadawais/create-guten-block)

I've released [create-guten-block](https://github.com/ahmadawais/create-guten-block) — a #0CJS toolkit for building and prototyping #Gutenblocks for WordPress. If the name `Gutenberg` was not there, I had to call it what? `create-wp-block` or `create-wp-plugin` — both of which are overly vague. 

While building something new, I want the audience to know what we have here — or the relation to what I have built with WordPress. 

>I think removing the name `Gutenberg` would be like removing the name `WooCommerce` since `WordPress eCommerce` could be more easy for users to understand what that is, but wait that's also a name of another plugin. Guess what there's a plugin called `Blocks` and many page builders that claim to be `blocks builders`.

#### ⚛ Example #2 React Ecosystem

Since `Gutenberg` makes use of the `React` JavaScript framework, maybe we can take inspiration from them. So, in the React ecosystem — they don't keep cooking everything under one single name called React. They name it and build a new ecosystem of their sufficiently big tool. So [many of them](https://code.facebook.com/projects/web/
).

- ✅ That's why you see, `GraphQL` could have easily been `ReactQL`. But they built `GraphQL` and then `graphql-js` and used it with `React. 
- ✅ Same is the story with `jest` — you see it's not `React Test` or something — but a more module painless JavaScript testing tool — that's not only used with React but in a number of other projects.

Same is the case here. While building [create-guten-block](https://github.com/ahmadawais/create-guten-block) I dug deep in the JavaScript community and what looked like a giant mess from the surface — felt like a lots of decision making — sometimes done right and other times wrong — but for the purpose of making devs life easier — which in turn is bringing more devs to the JavaScript world. 

🛠 I built a monorepo with [create-guten-block](https://github.com/ahmadawais/create-guten-block) and as of now the repo has four packages that can exist in any WordPress and two in even simple JavaScript projects that aren't related to WordPress at all. This has led to strange but pretty good stats. Many projects are using parts of my code in their projects not related to WordPress at all or without having to include it all.

I think what I am trying to say here is that `Gutenberg` is a big bold step of WordPress and we shouldn't strip off of its name — just because that's what we have always done. This jump towards the JavaScript community can actually help WordPress grow as a product, community, and we can strike for that 50% goal @m talks about.

🤞 But hey, this is just my opinion— I vote for the `Gutenberg` name to stay and get strong. The new themes and plugins that support `Gutenberg` should mention as such. It's a new bold WordPress without having to fork it all.

Peace! ✌️

**Comment by mor10** [comment]
@ahmadawais The "Gutenberg" name is not descriptive of anything and is already an established name for something totally different. Both "React" and "GraphQL" are descriptive names that carry some meaning and reference to what they do. "Gutenberg" does not. "WordPress Blocks" does.

**Comment by ahmadawais** [comment]
@mor10 I'll have to disagree with the narrative that names should be descriptive. That's not always true. E.g. my the name `jest` means `a thing said or done for amusement; a joke.` but that's a Unit Testing framework — that my friend is no joke. 😂

I think whatever sticks is a good name. I'm happy with a less verbose but more common name than the other options we have :)

**Comment by dmhendricks** [comment]
Perhaps the API can be referenced as Gutenberg but users will simply create "blocks." Sort of like Cocoa on OS X. Users don't know what Cocoa is, but developers do. Your point about it already being established for something is valid, but I'm not sure of any sort of infringement.

**Comment by kevinwhoffman** [comment]
One aspect that hasn't been touched on yet is how much we've stressed since WCUS that Gutenberg is "more than the editor." If we agree that the long-term Gutenberg vision goes beyond the editor (which is the official stance outlined in the project's documentation), then we should not further confusion by treating the term "Gutenberg" as synonymous with the editor in 5.0.

With that said, I'm in favor of "block editor" and "classic editor."

**Comment by JJJ** [comment]
Site editor vs. Content editor.

(The disconnect/miss is that the Customizer controls are on the left, and the Gutenberg controls are on the right, and they use different APIs, to do eventually the same things 2 years from now.)

I think it had to be just “the editor” with a classic “mode” or an “original flavor” or something.

**Comment by ChrisEdwardsCE** [comment]
I agree if a name change is to happen it should happen now. WordCamps and WordPress Meetups are holding sessions and talks on "Gutenberg" and if it were made a drinking game to take a shot whenever "Gutenberg" is said at our last few meetups, none of us would been able to walk out of the venue. 

Marketing & education wise the name is getting out there. With this being pushed out in the upcoming months, everyone that does any kind of WordPress event is doing something to educate the general public. Calling it by what it is now will make less confusion when it rolls out :)

**Comment by zgordon** [comment]
So it sounds like *most* folks here encourage this movement.

Can anyone pull in the team actually working on the project and decision makers who can speak to the possibility of doing things like changing the name of the plugin or the language in the handbook? I know this is something they have thought about as well so they likely have some good feedback. 

Although I hear a lot of folks saying we need to do this now, I think, practically it makes more sense to do this when GB rolls into Core. Right now it's not really accurate for me to say here is a course on the new editor in WordPress when it's not actually in WordPress yet and the plugin and docs all say Gutenberg. 

If we were to try to do this before GB goes into Core I feel a lot more with the plugin and docs would have to change. However, if we plan this for when it rolls into Core, some of those changes would happen automatically (like not needing to install a plugin named Gutenberg and hopefully the docs would drop the name as well since it is now just the core editor).


Also, belive the naming has already been set for us. There is the Editor (GB) and the Classic Editor (what we have now). 

Hoping to hear what Core devs and decision makers think on this!!

**Comment by khromov** [comment]
I disagree that the name should be dropped. Most people refer to TinyMCE as "the editor" right now, making Gutenberg "the new editor" would be very confusing. That phrasing could easily refer to another editor, like any number of available page builders. And there's now actually going to be two editors baked-in, so separating them is even more important. 

The only thing that detracts me from this idea is that the brand has been somewhat tarnished due to the less-than-stellar handling of the community during its development, but I've certainly felt that the conversation is pivoting into a more positive direction. (A less generous reading would be that people are getting out of the denial phase.)

Like it or not, Gutenberg *is* already a brand. WordPress.tv has 10 sessions on it and Google serves up 7 200 000 results for "gutenberg" + "wordpress". Destroying that work would serve no purpose - I truly don't understand the points @mor10 is making. "Clear language matters." Then call it the "Gutenberg block editor".

**Comment by carlhancock** [comment]
@khromov Gutenberg isn't a brand.  It's a project codename for a piece of functionality in WordPress. A key piece of functionality at that.  Once Gutenberg is the editor in WordPress it should simply be called WordPress.  We shouldn't continue to call it by some codename that most users aren't going to know or understand.

I don't care how many sessions are on WordPress.tv or how many results Google serves up.  The average user isn't looking at any of that.  They aren't browsing Github.  They aren't using the Gutenberg plugin.  They aren't perusing WordPress.tv and they aren't attending WordCamps.  To them WordPress is simply the software they are using to create their web site, the software their web developer or agency used to create their web site, or the software their hosted service used to create their web site. And they certainly won't be aware of any drama related to Gutenberg's development.

This actually illustrates why it can be counterproductive to use a codename for a project like this. The fact that there is even a discussion about continuing to call it Gutenberg or not is silly.  Developers can call it whatever they want.  To the users it's simply going to be WordPress just like it is now.

**Comment by kevinwhoffman** [comment]
> I disagree that the name should be dropped. Most people refer to TinyMCE as "the editor" right now, making Gutenberg "the new editor" would be very confusing.

The fact that most people refer to TinyMCE as "the editor" is precisely why we should refer to the new block editor in the same light. They are already familiar with the terminology and will understand what part of WordPress is changing.

**Comment by mikeschinkel** [comment]
Much thread.
Very opinions.
So bikeshed.
Wow.

**Comment by mor10** [comment]
@mikeschinkel The end-user cares more about where to find the instructions for how to use the advanced cloud-based bikeshed locks than how the powerplant runs. 

@khromov Branding is vital. That's why I'm bringing this up. A final decision needs to be made about how this feature will be referred to in the future. It is either "Gutenberg" everywhere or "[insert other name]" everywhere. Unlike TinyMCE which is a 3rd party solution with its own identity, Gutenberg is / will be a core feature of WordPress itself that lives inside WordPress. Giving it a non-descriptive name at that point goes against the design principles of WordPress itself. Post Formats are not called "Dewey" for this very reason. Once this thing is built into core, there is no need for a custom branded name. There will only be one editor - the block editor - and all documentation will focus on either that editor or the blocks it manages. The name "Gutenberg" ceases to play the role of explaining "this, not that" and instead takes on the role of confuser.

**Comment by mikeschinkel** [comment]
@mor10 My comment was more about the nature of the debate than what was being debated.

**Comment by JJJ** [comment]
Collectively, we have all answered our own questions without answering each other’s.

If the plan is to continue developing Gutenberg as an editor that can be used outside of WordPress core, the decision on what to name it isn’t up to us, and was already decided before development started.

Gutenberg will just live on as a separate React wrapper for a block based editor library that also happens to use TinyMCE for some blocks, and WordPress will just happen to adopt it instead of any of the other existing page-customizer type options.

In that scenario, the fact that Gutenberg even comes with a PHP layer or (WordPress support at-all) should be abstracted away eventually, in the way TinyMCE shouldn’t come with WordPress-specific code.

**Comment by scallemang** [comment]
I think we can have our cake and eat it too. E.g., from an imaginary rollout: "_Announcing Gutenberg – the new WordPress editor._ Gone is the WYSIWYG! The new system is all about blocks: blocks of text, blocks of images. [. . .]" Hell, if people can wrap their heads around—and pronounce—"WYSIWYG", I think things will work out. The name Calypso hasn't caused any pain so far, right? (Maybe I'm wrong? Is it divisive, or has it had any marketing issues?) It's just the dashboard to 90% of end-users, right?

iOS updates are called and marketed as Mountain Lion or High Sierra without any big fallout, I don't think, as non-descriptive as those may be. (Indeed, sometimes I run into update-specific issues with, like, Xcode compatibility and it's actually _helpful_ to have the marketed name for my Stack Overflow searches.)

All of which is to say, I'm enthusiastic for a scenario in which _Gutenberg_ is the product/feature name of the core WP editor where you work in _blocks_.

**Comment by maximebj** [comment]
Keep the name ! 

Sub branding is not a bad thing. 
This project is big enough to deeply change WordPress and deserves its own name

**Comment by karmatosed** [comment]
As this is a discussion, just removing the 'needs decision' from this for now.

**Comment by bjkeeton** [comment]
I am all about keeping the name. Between Google results, the internal WP community already knowing it, and the code referring to it, there's no reason not to keep it. I think @khromov has the right idea when referring to the "Gutenberg block editor" or something similar.

There isn't a reason why the branding can't still exist with the future of the project, code name or not. To say that the name should be deprecated because it serves no function totally disregards the work the WP community as a whole has done to accept it.

Even though tinyMCE is a third party software, people just call it the editor. The same is gonna be said of Gutenberg, too, in a few years. It seems like this is a semantic argument to me that's more about marketing and branding than anything, and the marketing and branding already exists. Taking GB out of the title (rather than adding clarity to it) is counterproductive.

Yes, Morten is right: Gutenberg on its own is not good. Gutenberg Block Editor is. Even just the Gutenberg Editor.

Regardless...you can't say that Gutenberg on its own doesn't hold meaning. It is inherently associated with revolutionary publishing, whether it has the word "editor" around it or not. People know it has to do with words, publishing, writing, etc. Even if we called it just Gutenberg WP, it'd be at least connoted, if not clear.

**Comment by transl8or** [comment]
My spontaneous idea about that issue would be to find a better word for "**Editor**". 
The word Gutenberg can easily vanish then when the merge into core begins and when we have something better for 'The New Editor' or 'The Block Editor'. 

Think **Customizer**! How was it back then?

And as someone already said, I would also prefer to think functional. 
I know normal users, who still have trouble to understand what an Editor is. They start thinkin' about Microsoft Word and stuff.

The task for WordPress, in the context of democratizing publish, is simply to create content to publish. 
Or as also already said: People create their Websites with a Blog or a Blogs alone with WordPress.
 
[BTW I also see the word 'Block' a bit critical for future in the long run as it is so close to the sound of the word 'Blog' or even 'Black'. Accessibility? Can you hear me?!]

Now, has anyone feedback on the idea, to call the new Editor coming within, as part of WordPress 5.0 something different? Maybe... 

- Creatr
- Formator
- Constructr

... for content made of rich media to publish in an even better web.

**Comment by bjkeeton** [comment]
@transl8or I could see it being called the Publisher, along those lines. Keeping it away from editor (which actually already exists in **Appearance -> Editor** as something totally different.

**Comment by transl8or** [comment]
@bjkeeton 'Publisher', that's also a good one :+1:

**Comment by mellis32** [comment]
Throwing my opinion in the ring as a relatively new user:
I support depreciating the Guttenberg code name.

Please no more crazy gimmicky or proprietary names for the editor! If Guttenberg really will take over all of Wordpress, then it should just be called the Wordpress editor.

Please no proprietary names like Gutenberg, or TinyMCE, **this only adds confusion for new users and makes Wordpress seem less approachable and less familiar than other forms of writing and composing.**

and especially please no gimicky internet name like Creatr, Formator, Constructr, Publishr, ect.

**Comment by danieltj27** [comment]
Just throwing my opinion in here; I'm for dropping the project codename and referring to it as non-descriptively as `a/the content editor` with a block mode and classic mode.

**Comment by wpmark** [comment]
It would seem to me that change the name of "Gutenberg" now to something else would mean the need to change all of the function and hooks names in the plugin (eventually in core).

It occurs to me that there are many plugins in the wordpress.org repository now that will be actively using these filter and action names. If these get change won't all these plugins break?

One such example of a recent plugin mention is this one: https://wordpress.org/plugins/manager-for-gutenberg/

The plugin above and many more like it would break if all the names are changed?

**Comment by danieltj27** [comment]
I’ve changed my opinion on this. After reading what people have said, I think Gutenberg has because a very important point in WordPress’ history and so keeping the name is important; as well as the code implications of course.

**Comment by teamcoltra** [comment]
I am commenting to add that I don't like the name because "Project Gutenberg" is already a thing and so searching for Gutenberg Project and Project Gutenberg is only going to be bad for SEO, or if this takes off in popularity it may drown out the actual Project Gutenberg which is dedicated to an incredibly important task.

Not only that but you create naming conflicts that deal with the other Gutenberg related code.

I'm not sure how to fix this problem now, I came in too late to voice this opinion sooner... but consider me a person who doesn't like this.

**Comment by pento** [comment]
The Gutenberg name will continue to live on as the label for the overarching Gutenberg project: Gutenberg phase 1 (the block editor) is approaching its conclusion, while Gutenberg phase 2 (site customisation) will start to ramp up soon.

There's no immediate need to deprecate the Gutenberg name (if there ever will be), so I'm going to close this issue. We can always revisit it at the conclusion of the Gutenberg project, but there's no timeline for that at this stage.
